demo/grid/index.html

```diff
@@ -0,0 +1,165 @@
+        All classes are consistent with the
+        <a href="https://getbootstrap.com/docs/4.0/layout/grid/">Bootstrap Grid Documentation.</a>
+      </p>
+      <h2 class="u-gamma u-mb">Mobile-Responsive Columns</h2>
+      <div class="container-fluid debug u-mb-xl">
+        <div class="row">
+          <div class="col-md">
+            one of three columns
+          </div>
+          <div class="col-md">
+            two of three columns
+          </div>
+          <div class="col-md">
+            three of three columns
+          </div>
+        </div>
+      </div>
+      <h2 class="u-gamma u-mb">RTL Locales</h2>
+      <p class="u-mb">
+        You can enable RTL layouts by using the <code class="c-code">.is-rtl</code> class with the
+        <code class="c-code">.container</code> or <code class="c-code">.container-fluid</code>
+        wrapper classes</a>.
+      </p>
+      <div class="container-fluid is-rtl debug u-mb-xl">
+        <div class="row">
+          <div class="col-md">
+            one of three columns
+          </div>
+          <div class="col-md">
+            two of three columns
+          </div>
+          <div class="col-md">
+            three of three columns
+          </div>
+        </div>
+      </div>
+      <h2 class="u-gamma u-mb">Setting One Column Width</h2>
+      <div class="container-fluid debug u-mb-xl">
+        <div class="row">
+          <div class="col">
+            1 of 3
+          </div>
+          <div class="col-6">
+            2 of 3 (wider)
+          </div>
+          <div class="col">
+            3 of 3
+          </div>
+        </div>
+        <div class="row">
+          <div class="col">
+            1 of 3
+          </div>
+          <div class="col-5">
+            2 of 3 (wider)
+          </div>
+          <div class="col">
+            3 of 3
+          </div>
+        </div>
+      </div>
+      <h2 class="u-gamma u-mb">Variable Width Content</h2>
+      <div class="container-fluid debug u-mb-xl">
+        <div class="row justify-content-md-center">
+          <div class="col col-lg-2">
+            1 of 3
+          </div>
+          <div class="col-md-auto">
+            Variable width content
+          </div>
+          <div class="col col-lg-2">
+            3 of 3
+          </div>
+        </div>
+        <div class="row">
+          <div class="col">
+            1 of 3
+          </div>
+          <div class="col-md-auto">
+            Variable width content
+          </div>
+          <div class="col col-lg-2">
+            3 of 3
+          </div>
+        </div>
+      </div>
+      <section id="playground">
+        <h1 class="c-main__subtitle u-alpha u-mb">Playground</h1>
+        <div class="c-playground">
+          <div class="c-playground__preview"></div>
+<textarea class="c-playground__code">
+<div class="container-fluid debug">
```

Comment:
make this section "spaceless" with respect to the `<textarea>`. No leading/trailing whitespace. See other demo pages for details.

Reply:
Fixed.